docs/spec-find.md

```diff
@@ -1,7 +1,84 @@
 ---
 id: spec-find
-title: spec-find
-sidebar_label: spec-find
+title: Find Operation
 ---
 
-spec-find TODO
+## Find multiple objects Specification
+
+Find operation can be respresented as follows
+
+```graphql
+type Query {
+  findNotes(filter: NoteFilter, orderBy: OrderByInput): [Note]!
+}
+```
+
+Finding objects can be done by creating numerous helpers.
+Filter input type:
+```graphql
+input NoteFilter {
```

Comment:
I think the example model uses should have a field with each of the common scalar types, ID, Int, Float, Boolean, String.